build-support/build-opts.js

```diff
@@ -23,7 +23,12 @@ const replacementStrings = {
     }
   ],
   css: [],
-  js: []
+  js: [
+    {
+      search: /__PACKAGE_VERSION__/g,
```

Comment:
The version replacement in the CFM metadata appears to have been broken since the gulp => webpack transition.